### PR TITLE
change shipping title based on country user is from

### PR DESCRIPTION
**Pull request by btkostner**
### Changes Summary

- Changes the 'Worldwide Shipping' title based on the country the user is from

https://beta.elementary.io/store-shipping/store/

### CODE REVIEW

File: store/index.php [hunk 1]
```diff
@@ -144,7 +145,19 @@
...
+            $country_code = getCurrentCountry($ip);
+            $shipping_text = 'Worldwide Shipping';
+
+            if ($country_code && $country_code !== 'US') {
+                $country_name = getCurrentCountryName($ip, $page['lang']);
+
+                if ($country_name) {
+                    $shipping_text = 'Now shipping to ' . $country_name;
+                }
+            }
```
Comment (lewisgoddard) [inline store/index.php]: You need an else here to handle Cuba, Iran, and North Korea.

**Comment by lewisgoddard** [comment]
Code looks fine, but the beta system doesn't seem to run.

**Comment by lewisgoddard** [comment]
> Now shipping to United Kingdom
Hm, really it is "The United Kingdom"

**Comment by lewisgoddard** [comment]
There's quite a few with that sort of exception, does that database not have a nice-name field? https://en.wikipedia.org/wiki/List_of_sovereign_states

**Comment by btkostner** [comment]
No it does not. It's all translated fields.

**Comment by btkostner** [comment]
Final thoughts on this. Should we setup a catch all for countries that need "the" in front of the name, or just role as is right now?

**Comment by lewisgoddard** [comment]
I'd appreciate the catch-all, but this isn't going to be easily translatable, is it?

**Comment by btkostner** [comment]
It's currently using translations from the library.

Translations for Japan:
```
{
["de"]=> string(5) "Japan"
["en"]=> string(5) "Japan"
["es"]=> string(6) "Japón"
["fr"]=> string(5) "Japon"
["ja"]=> string(6) "日本"
["pt-BR"]=> string(6) "Japão"
["ru"]=> string(12) "Япония"
["zh-CN"]=> string(6) "日本" 
}
```

**Comment by lewisgoddard** [comment]
Sure then, we should try and catch those exceptions and see whether or not that seems better once translated.

**Comment by btkostner** [comment]
I'm going to close this PR because translations are going to be a huge pain for very little gain.

**Pull request closed without merging**